Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115.py

```diff
@@ -807,15 +812,47 @@ def endpoint_isolation_remove_command(client: Client, args: Dict[str, Any]) -> C
     return command_results
 
 
-def fetch_incidents(client: Client) -> list:
+def fetch_alerts_related_incident(client: Client, incident_id: str) -> list[dict[str, Any]]:
+    """
+    Returns the alerts that are associated with an incident.
+    """
+    alerts = []
+    has_next = True
+    page_number = 0
+
+    while has_next:
+        try:
+            response_body = client.incident_list_alerts_request(
+                page_number=str(page_number),
+                id_=incident_id,
+                page_size=None
+            )
+            alerts.extend(response_body.get('items', []))
+            has_next = response_body.get('hasNext', False)
+            page_number += 1
+        except DemistoException:
+            demisto.error("Error occurred while fetching alerts.")
+            raise
+    return alerts
+
+
+def fetch_incidents(client: Client, params: dict) -> list:
     total_items, last_fetched_ids, timestamp = client.get_incidents()
     incidents: List[Dict] = []
     new_ids = []
     for item in total_items:
         inc_id = item.get('id')
         new_ids.append(inc_id)
         item['incident_url'] = client.get_incident_url(inc_id)
-        incident = {"name": f"RSA NetWitness 11.5 {item.get('id')} - {item.get('title')}",
+
+        # add to incident object an array of all related alerts
+        if argToBoolean(params.get('import_alerts')):
+            item['alerts'] = fetch_alerts_related_incident(client, item.get('id'))
+            if alerts := item.get('alerts'):
```

Comment:
why assign to item, and then get? Consider saving `fetch_alerts_...` to a variable instead.